aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -1647,5 +1658,324 @@ Tensor min_mps(const Tensor& input_t) {
     return min_max_mps(input_t, dim, keepdim, MPSReductionType::MIN, "min_mps");
 }
 
+static bool mpsSupportsSort() {
```

Comment:
Instead of this @malfet added an option to Query [MacOS 13+](https://github.com/pytorch/pytorch/pull/88529/files#diff-c7bb5bafa54cf2dfc86cafb769b80f8a8dfbf3342020f11b1b3fae507b56fef2R93). Can you use this instead? These are supported in MacOS Ventura.

Reply:
Yes removed this